roles/memcached/tasks/main.yml

```
@@ -1,21 +1,21 @@
 ---
 # Include variables and define needed variables.
 - name: Include OS-specific variables.
-  include_vars: "{{ ansible_os_family }}.yml"
+  ansible.builtin.include_vars: "{{ ansible_os_family }}.yml"
 
 - name: Define memcached_user.
-  set_fact:
+  ansible.builtin.set_fact:
     memcached_user: "{{ __memcached_user }}"
   when: memcached_user is not defined
 
 # Setup/install tasks.
 - name: Install Memcached.
-  package: name=memcached state=present
+  ansible.builtin.package: name=memcached state=present
```

While you're updating this role, it would be great to use "new" syntax for all tasks. This isn't something we've discussed before . . . but if you look at the next task in this file, you'll see that all the parameters are on separate lines below the module name, indented two spaces, and use `:` instead of `=` as the separator. This is "modern" (since I think Ansible 2.0, maybe earlier) syntax, and I find it more readable. For this task, the update would look like this:
```suggestion
  ansible.builtin.package:
    name: memcached
    state: present
```